File: Lab 1 - NguyenDucKhai/C++/Lab 1/Rectangle/rectangle.cpp
#ifndef RECTANGLE_H
#define RECTANGLE_H
#include <iostream>

using namespace std ;

class Rectangle
{
	public:
	
	Rectangle();
	Rectangle(double, double) ;
	
	~Rectangle();
	
	void setLength(double);
	void setWidth(double) ;
	
	double getLength() ;
	double getWidth() ;
	double getArea() ;
		
		
	private:
		double length, width;
		
}

Rectangle::Rectangle()
{
	width = 1 ;
	length = 1 ;
}

Rectangle::Rectangle(double l, double w)
{
	width = w ;
	length = l ;
}
Rectangle::setLength(double l)
{
	length = l ;
}
Rectangle::getLength()
{
	return length ;
}
Rectangle::setWidth(double w)
{
	width = w;
}
Rectangle::getWidth()
{
	return width;
}
Rectangle::getArea()
{
	return length * width ;
}

File: Lab 1 - NguyenDucKhai/Java/Rectangle/src/Rectangle.java

public class Rectangle {
	private double width, length;
	public 
	void setRectangle(double l, double w)
	{
		width = w;
		length = l;
	}
	double getWidth()
	{
		return width;
	}
	double getLength()
	{
		return length;
	}
	double getArea()
	{
		return width * length;
	}
	
	public static void main(String args[])
	{
		Rectangle R1 = new Rectangle();
		R1.setRectangle(10, 5);
		System.out.println("Width: " +R1.getWidth());
		System.out.println("Length: " +R1.getLength());
		System.out.println("Area: " +R1.getArea());
		
		
	}
}
